libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/StreamingExtensions/BotFrameworkStreamingExtensionsAdapter.cs

```diff
@@ -0,0 +1,205 @@
+        /// <remarks>If the activities are successfully sent, the task result contains
+        /// an array of <see cref="ResourceResponse"/> objects containing the IDs that
+        /// the receiving channel assigned to the activities.</remarks>
+        /// <seealso cref="ITurnContext.OnSendActivities(SendActivitiesHandler)"/>
+        public override async Task<ResourceResponse[]> SendActivitiesAsync(ITurnContext turnContext, Activity[] activities, CancellationToken cancellationToken)
+        {
+            if (turnContext == null)
+            {
+                throw new ArgumentNullException(nameof(turnContext));
+            }
+
+            if (activities == null)
+            {
+                throw new ArgumentNullException(nameof(activities));
+            }
+
+            if (activities.Length == 0)
+            {
+                throw new ArgumentException("Expecting one or more activities, but the array was empty.", nameof(activities));
+            }
+
+            var responses = new ResourceResponse[activities.Length];
+
+            /*
+             * NOTE: we're using for here (vs. foreach) because we want to simultaneously index into the
+             * activities array to get the activity to process as well as use that index to assign
+             * the response to the responses array and this is the most cost effective way to do that.
+             */
+            for (var index = 0; index < activities.Length; index++)
+            {
+                var activity = activities[index];
+                var response = default(ResourceResponse);
+                _logger.LogInformation($"Sending activity.  ReplyToId: {activity.ReplyToId}");
+
+                if (activity.Type == ActivityTypesEx.Delay)
+                {
+                    // The Activity Schema doesn't have a delay type build in, so it's simulated
+                    // here in the Bot. This matches the behavior in the Node connector.
+                    var delayMs = (int)activity.Value;
+                    await Task.Delay(delayMs, cancellationToken).ConfigureAwait(false);
+
+                    // No need to create a response. One will be created below.
+                }
+                else if (activity.Type == ActivityTypesEx.InvokeResponse)
+                {
+                    turnContext.TurnState.Add(InvokeReponseKey, activity);
+
+                    // No need to create a response. One will be created below.
+                }
+                else if (activity.Type == ActivityTypes.Trace && activity.ChannelId != "emulator")
+                {
+                    // if it is a Trace activity we only send to the channel if it's the emulator.
+                }
+
+                var baseUrl = activity.ServiceUrl + (activity.ServiceUrl.EndsWith("/") ? string.Empty : "/");
+                var requestPath = $"{baseUrl}v3/conversations/{activity.Conversation.Id}/activities/{activity.Id}";
```

Comment:
You and Joe need to come up with the correct request path. My suggestion is to NOT include the baseURL but instead just make it the path from the root (it's always to the same host). So this would become:
"/v3/conversations/{conversationId}/activities/{activityId}"

Reply:
Updated bot side to the example Jeff gave, need to coordinate with Joe to make sure we're compatible.